### ISSUE TITLE
Bug in catalog submission

### ISSUE THREAD

**Issue by jbusecke** [leap-stc/data-management#42]
@andersy005 I just tried to expose an existing dataset on our buckets via the catalog in #39.
All the CI [succeeded](https://github.com/leap-stc/data-management/actions/runs/5740234945) but I afterwards noticed that I added the file in the wrong directory
<img width="252" alt="image" src="https://github.com/leap-stc/data-management/assets/14314623/5e5dd71d-de3a-45ca-8b88-886d59f787a8">
[repo at the corresponding ref](https://github.com/leap-stc/data-management/tree/a76e0690655b29f57160b76271111d88843d51c3). 

I moved the file and again the CI [succeeded](https://github.com/leap-stc/data-management/actions/runs/5740303472), but the new dataset is not shown in the [catalog](https://leap-data-catalog.vercel.app).

My immediate question is how to fix the addition to the catalog, but I am also wondering if this case of mistake could be caught in the CI.

**Comment by jbusecke** [comment]
Oh I just saw #41 pop up and realized I did not understand this mechanic properly. The second question remains though, and I wonder if this pattern is documented somewhere? Or should we add some docs to the README atleast?

**Comment by jbusecke** [comment]
Ill merge this now (since some folks need this dataset), but I think this is still a bug worth investigating.

**Comment by cisaacstern** [comment]
@andersy005 @jbusecke so, how exactly does one make a PR to add a dataset to the catalog? I'm getting close on the finishing the ClimSim datasets (or at least one of them), but am not clear how I should expose these in the catalog.

**Comment by jbusecke** [comment]
I had a chat with @andersy005 and @norlandrhagen the other day about this. I think ultimately we want a final 'cataloging' stage for all our recipes, but this dataset is a great example of where we DO NOT want to rerun this if not necessary (its expensive and takes many days to run). So my proposal is to first focus on a manual submission. I guess this should be mostly docs at this point?

**Comment by cisaacstern** [comment]
Are there any current docs for manual submission?

**Comment by jbusecke** [comment]
Nope. Is that something that you could work on @andersy005 and @norlandrhagen?

**Comment by andersy005** [comment]
@jbusecke, i certainly intend to work on the docs sometime this week.

**Comment by andersy005** [comment]
@cisaacstern, apologies for not getting back to you sooner :( this slipped through the cracks. i've just merged a PR including some documentation on how to add datasets to the catalog:  https://github.com/leap-stc/data-management/blob/main/catalog/README.md. let me know if there are any missing parts or if anything needs further clarification.

**Comment by cisaacstern** [comment]
Thanks @andersy005! I will take a look.

**Comment by jbusecke** [comment]
I have split out a discussion about the multi PR structure in #65 and would close this issue now unless others object?
